docs/userGuide/contentAuthoring.md

```diff
@@ -576,4 +576,12 @@ Note:
 - Footers should not be nested in other components or HTML tags. If found inside, they will be shifted outside to be rendered properly.
 - [MarkBind components](#use-components) and [includes](#include-contents) are not supported in footers.
 
-</div>
+### Using Anchors
+
+Sometimes, you may want a user of your site to be able to obtain a link to a heading within a page, to share with someone else for example.
+
+MarkBind can automatically generate heading anchors for your page.

```

Comment:
_Nit:_
MarkBind ~~can~~ automatically generate**s** heading anchors for your page.

Reply:
Also might be better to show an example. But we don't need a picture, just ask the reader to try it out themselves with the `### Using Anchors` heading above.

Reply:
Sure, but I don't think the last one is a good example because it's at the bottom of the page - the screen doesn't move noticeably